hamilton/plugins/pandas_extensions.py

```diff
@@ -983,6 +984,137 @@ def load_data(self, type: Type) -> Tuple[DATAFRAME_TYPE, Dict[str, Any]]:
     @classmethod
     def name(cls) -> str:
         return "feather"
+    
+@dataclass
+
+class PandasTableReader:
+    path_or_buffer: str
+    sep: str = "\t"
+    delimiter: str = None
+    header: str = "infer"
+    names: str = None
+    index_col: str = None
+    usecols: str = None
+    engine: str = "c"
+    converters: str = None
+    true_values: str = None
+    false_values: str = None
+    skipinitialspace: bool = False
+    skiprows: str = None
+    skipfooter: int = 0
+    nrows: int = None
+    na_values: str = None
+    keep_default_na: bool = True
+    na_filter: bool = True
+    verbose: bool = False
+    skip_blank_lines: bool = True
+    parse_dates: bool = False
+    infer_datetime_format: bool = False
+    keep_date_col: bool = False
+    date_parser: str = None
+    dayfirst: bool = False
+    cache_dates: bool = True
+    iterator: bool = False
+    chunksize: int = None
+    compression: str = "infer"
+    thousands: str = None
+    decimal: str = "."
+    lineterminator: str = None
+    quotechar: str = '"'
+    quoting: int = 0
+    doublequote: bool = True
+    escapechar: str = None
+    comment: str = None
+    encoding: str = None
+    dialect: str = None
+    error_bad_lines: bool = True
+    warn_bad_lines: bool = True
+    delim_whitespace: bool = False
+    low_memory: bool = True
+    memory_map: bool = False
+    float_precision: str = None
+    storage_options: str = None
+
+    reader_name: ClassVar[str] = "feather"
+
+    @classmethod
+    def name(cls) -> str:
+        return cls.reader_name
+
+    def _get_loading_kwargs(self) -> Dict[str, Any]:
+        kwargs = {
```

Comment:
this does not look like the other classes. It should look something [like this](https://github.com/DAGWorks-Inc/hamilton/blob/main/hamilton/plugins/pandas_extensions.py#L265-L301).